src/lazy/binary/raw/v1_1/immutable_buffer.rs

```diff
@@ -505,8 +505,7 @@ impl<'a> ImmutableBuffer<'a> {
             }
             Ok((None, after_nops))
         } else {
-            self.read_value(opcode)
-                .map(|(v, after)| (Some(v), after))
+            self.read_value(opcode).map(|(v, after)| (Some(v), after))
```

Comment:
🗺️ `cargo fmt` re-flowed some things in this file even though they hadn't changed in this commit. You can ignore this and the one below (lines 630-654).